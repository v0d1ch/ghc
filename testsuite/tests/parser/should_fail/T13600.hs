module T13600 where

f4 :: [Int] -> IO Int
f4 !x = return (sum x)
