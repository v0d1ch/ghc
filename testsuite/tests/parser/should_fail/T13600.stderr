The type signature for ‘f4’ lacks an accompanying binding
